ospatch/updates_linux.go

```diff
@@ -92,10 +94,49 @@ func systemRebootRequired() (bool, error) {
 	return false, fmt.Errorf("no recognized package manager installed, can't determine if reboot is required")
 }
 
+func updatePackages(r *patchRun) error {
+	var errs []string
+	if packages.AptExists {
+		var opts []packages.AptGetUpgradeOption
+		if r.Job.ReportPatchJobInstanceDetailsResponse.PatchConfig.Apt != nil {
```

Comment:
Can this be combined with `if packages.AptExtists && ...` to remove an extra level of nesting? (Same for Yum & Zypper).

Reply:
APT existing on the system means we will patch APT, that doesn't mean there are any settings under PatchConfig.Apt, so it could be nil

Reply:
Ah, this is only for adding additional args. Got it :+1: